src/main/java/com/dougnoel/sentinel/elements/tables/Table.java

```diff
@@ -478,6 +478,16 @@ public void clickElementInRowThatContains(int ordinalRow, By elementToClick) {
 		getElementInRowThatContains(ordinalRow, elementToClick).click();
 	}
 
+	/**
+	 * Clicks the cell found at the given x,y coords in the table.
+	 * @param column int the x coord of the cell in the table
+	 * @param row int the y coord of the cell in the table
+	 */
+	public void clickElementAtCoord(int column, int row) {
+		WebElement tableRow = getOrCreateRowElements().get(row);
+		WebElement cell = tableRow.findElement(By.xpath(".//" + tableCellDataTag + "[" + column + "]"));
```

Comment:
This xpath might want to be a const at the header so someone can easily extend it

Reply:
I don't believe we can actually make it a constant as it's being modified, but I'll make note of it in the header that it's expected and behaving with it in that form

Reply:
Nevermind!

Reply:
Still could use input on it.